src/v3/src/transformer/webauthn/transformWebAuthNAuthenticator.ts

```diff
@@ -187,8 +191,20 @@ export const transformWebAuthNAuthenticator: IdxStepTransformer = ({ transaction
     uischema.elements.unshift(submitButtonEle);
 
     appendViewCallouts(uischema, name, relatesTo);
+  } else {
+    infoBoxElement = {
+      type: 'InfoBox',
+      options: {
+        message: loc('oie.webauthn.error.not.supported', 'login'),
+        class: 'ERROR',
+        dataSe: 'callout',
+      },
+    };
   }
-  uischema.elements.unshift(informationalTextElement);
+  const informationalElements: UISchemaElement[] = [informationalTextElement, infoBoxElement]
+    .map((x) => x !== undefined && x)
+    .filter(Boolean as unknown as ExcludesFalse);
```

Comment:
Is there any way to avoid this by directly pushing/unshifting the elements onto the uischema.elments array?

If not, would this suffice?
```suggestion
  const informationalElements: UISchemaElement[] = [informationalTextElement, infoBoxElement]
    .filter((item) => typeof item !== 'undefined');
```

Reply:
The above didn't work, but I refactoring it since the original solution wasn't really necessary. 🙏